src/index.html

```diff
@@ -9,6 +9,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <div class="flex_container">
+        <a href="#home" class="logo_link">
+          <img
+            src="images/logo.png"
+            alt="logo"
+            class="logo flex_item"
+          >
+        </a>
+        <nav class="flex_item navigation">
+          <ul class="menu">
+            <li class="menu_item"><a href="*" class="nav_link is-active">Apple</a></li>
+            <li class="menu_item"><a href="*" class="nav_link">Samsung</a></li>
```

Comment:
Try to use BEM methodology, for example where nav it's block(parent), and ul, li, a are is elements(children)
```
<nav class='nav'>
   <ul class='nav__list'>
      <li class='nav__item'>
        <a class='nav__link'></a>
      </li>
   </ul>
<nav>
```